nixos/common.nix

```
@@ -81,6 +81,16 @@ in
       argument in Home Manager. This disables the Home Manager
       options {option}`nixpkgs.*`'';
 
+    backupCommand = mkOption {
+      type = types.nullOr types.str;
```

```suggestion
      type = types.nullOr (types.either types.str types.path);
```

Can support a command that's a path, too.